web/client/src/routes.tsx

```diff
@@ -5,14 +5,10 @@ import Loading from '@components/loading/Loading'
 import Spinner from '@components/logo/Spinner'
 import IDE from './library/pages/ide/IDE'
 
-const Editor = lazy(async () => await import('./library/pages/editor/Editor'))
-const Docs = lazy(async () => await import('./library/pages/docs/Docs'))
-const DocsContent = lazy(
-  async () => await import('./library/pages/docs/Content'),
-)
-const DocsWelcome = lazy(
-  async () => await import('./library/pages/docs/Welcome'),
-)
+const Editor = lazy(() => import('./library/pages/editor/Editor'))
```

Comment:
Curious why the async was removed.

Reply:
we had TS lint rule that forces to await all async functions. lazy already expects a Promise-like object and import returns promise, so no need to await for import because lazy should do it anyway. (https://react.dev/reference/react/lazy#reference)